appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -42,7 +42,8 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
                     ext.outputChannel.appendLog(creatingNewAppInsights);
                     progress.report({ message: creatingNewAppInsights });
 
-                    context.appInsightsComponent = await client.components.createOrUpdate(rgName, aiName, { kind: 'web', applicationType: 'web', location: appInsightsLocation });
+                    context.appInsightsComponent = await client.components.createOrUpdate(rgName, aiName, { kind: 'web', applicationType: 'web', location: appInsightsLocation, 
+                    workspaceResourceId: context.logAnalyticsWorkspace?.id});
```

Comment:
This is a strange spot for a newline

Reply:
Well, I agree, but the indentation and whatnot is weird here.

Reply:
![image](https://user-images.githubusercontent.com/5290572/180048776-6f9e912f-2110-4754-a981-ad82e187ec9a.png)
 I think it might just be a product of GitHub viewer

Reply:
I don't think it's GitHub. Arguments on subsequent lines should be indented; fields within objects that are themselves arguments should be indented further.

Like this:
```typescript
                    context.appInsightsComponent = await client.components.createOrUpdate(
                        rgName,
                        aiName,
                        {
                            kind: 'web',
                            applicationType: 'web',
                            location: appInsightsLocation, 
                            workspaceResourceId: context.logAnalyticsWorkspace?.id
                        }
                    );
```